### ISSUE TITLE
Authority overlay

### ISSUE THREAD

**Issue by talljoy** [thekesolutions/koha-plugin-lti-import#10]
Can this be used for authority overlays?  How much work involved?  There would need to be a second config file.

**Comment by tomascohen** [comment]
We can add the record type dropdown, to let users choose between biblios and auths. The question is (I guess) matching rules.

**Comment by talljoy** [comment]
The matching rule would be different.  And the overlay configuration would be different also.

Rule: Match on 010$a tag - This rule would need to be set up and specified in the configuration
Overlay: example: Virginia wants to match on 010$a and overlay all but the 690 tag.

**Comment by talljoy** [comment]
Yes, that is what I was thinking.  Can you give me an estimate on how long this would take before we decide to do it or not?

Thanks
joy

**Comment by tomascohen** [comment]
1 day
